dockerfile/cuda11.1.1.dockerfile

```diff
@@ -123,4 +123,5 @@ RUN make -j ${NUM_MAKE_JOBS} -C third_party cuda
 
 ADD . .
 RUN python3 -m pip install .[nvidia,torch,ort] && \
-    make cppbuild
+    make cppbuild && \
+    make postinstall
```

Comment:
Would you please add to other docker as well?

Reply:
sure, added